### DISCUSSION TITLE
PR review “Jump to file” list filtering

### DISCUSSION THREAD

**Discussion by adamdehaven** [community/community#41159] [Mobile]
It would be great if the "Jump to file" interface (shown during a code review on mobile) included a filename filter at the top.

The input would allow a user to enter case-insensitive text that would filter the list of files instead of having to scroll through the entire changed file list. (Bonus points if it accepts regular expressions or pattern match)

_I'm not requesting search; just list filtering_

![image](https://user-images.githubusercontent.com/2229946/206624665-51afe989-4a73-41da-9293-2773cbbeb958.jpeg)
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.92.0 (85285908)
Device: iPhone 12 (iOS 16.1.1)
</details>

**Comment by candyho** [comment]
Hey thanks for sharing the feedback! I've created an issue in our internal repo to track this request. Hopefully we can tackle this in the near future.
